.database_generator/mangos_translation/lightshope_merge.py

```
@@ -0,0 +1,201 @@
+  tables = cur.fetchall()
+  if not tables:
+    raise RuntimeError(f"No locales tables found in {sql_path}")
+
+  table_name = tables[0][0]
+  cur.execute("SELECT sql FROM sqlite_master WHERE type='table' AND name=?", (table_name,))
+  create_sql_row = cur.fetchone()
+  create_sql = create_sql_row[0] if create_sql_row else None
+
+  cur.execute(f"PRAGMA table_info({table_name})")
+  columns = [row[1] for row in cur.fetchall()]
+  cur.execute(f"SELECT * FROM {table_name}")
```

Ruff (0.14.6)

Possible SQL injection vector through string-based query construction

(S608)